test/quantization/test_quantized_op.py

```diff
@@ -2005,20 +2005,22 @@ def test_group_norm(self):
     @skipIfNoFBGEMM
     def test_instance_norm(self):
         max_sides = (4, 5)
-        side_lens = (2, 8, 11)
+        shape_list = ([2, 2, 2, 2], [8, 8, 8, 8], [11, 11, 11, 11], [1, 4, 224, 224, 160])
```

Comment:
hi @leslie-fang-intel , we are seeing our unit tests timeout on this PR when run in CI, probably because we are adding a high # of expensive use cases here because of all the combinations.  Any chance we could just add a single test case to test for overflow, without the various combinations?  Something like

```
shape_list = ([2, 2, 2, 2], [8, 8, 8, 8], [11, 11, 11, 11])
...
test_cases = itertools.product(*combined)
# add just one test case to test overflow
test_cases.append(
    [1, 4, 224, 224, 160],  # shape,
    torch.qint8,
    0.1,  # scale
    0,  # zero_point
    False,   # channels_last
    affine,  # affine
]
```

Reply:
@vkuzo done and rebased.